src/chmfile.cpp

```diff
@@ -521,11 +523,14 @@ bool CHMFile::BinaryIndex(CHMListCtrl& toBuild, const wxCSConv& cv)
 
                     offset += sizeof(uint32_t);
                     spaceLeft -= sizeof(uint32_t);
-                }
+                }                
```

Comment:
Please avoid making this type of change that only modifies whitespace. Running `clang-format` should help with that.